control-plane-operator/controllers/hostedcontrolplane/kubevirt_assets/assets.go

```diff
@@ -0,0 +1,1568 @@
+}
+
+func GenerateTenantNodeServiceAccountResources(namespace string) []crclient.Object {
+	mustacheRenderTemplate := map[string]string{"NAMESPACE": namespace}
+
+	infraServiceAccountBytes := getContents("tenant_node_serviceaccount.yaml")
+	renderedInfraServiceAccount := mustache.Render(string(infraServiceAccountBytes), mustacheRenderTemplate)
+	serviceaccount := &corev1.ServiceAccount{}
+	if err := yaml.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(renderedInfraServiceAccount)), 500).Decode(&serviceaccount); err != nil {
+		panic(err)
+	}
+
+	infraRoleBytes := getContents("tenant_node_clusterrole.yaml")
+	renderedInfraRole := mustache.Render(string(infraRoleBytes), mustacheRenderTemplate)
+	role := &rbacv1.ClusterRole{}
+	if err := yaml.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(renderedInfraRole)), 500).Decode(&role); err != nil {
+		panic(err)
+	}
+
+	infraRoleBindingBytes := getContents("tenant_node_clusterrolebinding.yaml")
+	renderedInfraRoleBinding := mustache.Render(string(infraRoleBindingBytes), mustacheRenderTemplate)
+	rolebinding := &rbacv1.ClusterRoleBinding{}
+	if err := yaml.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(renderedInfraRoleBinding)), 500).Decode(&rolebinding); err != nil {
+		panic(err)
+	}
+
+	return []crclient.Object{serviceaccount, role, rolebinding}
+}
+
+func GenerateTenantControllerServiceAccountResources(namespace string) []crclient.Object {
+	mustacheRenderTemplate := map[string]string{"NAMESPACE": namespace}
+
+	infraServiceAccountBytes := getContents("tenant_controller_serviceaccount.yaml")
+	renderedInfraServiceAccount := mustache.Render(string(infraServiceAccountBytes), mustacheRenderTemplate)
+	serviceaccount := &corev1.ServiceAccount{}
+	if err := yaml.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(renderedInfraServiceAccount)), 500).Decode(&serviceaccount); err != nil {
+		panic(err)
+	}
+
+	infraRoleBytes := getContents("tenant_controller_clusterrole.yaml")
+	renderedInfraRole := mustache.Render(string(infraRoleBytes), mustacheRenderTemplate)
+	role := &rbacv1.ClusterRole{}
+	if err := yaml.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(renderedInfraRole)), 500).Decode(&role); err != nil {
+		panic(err)
+	}
+
+	infraRoleBindingBytes := getContents("tenant_controller_clusterrolebinding.yaml")
+	renderedInfraRoleBinding := mustache.Render(string(infraRoleBindingBytes), mustacheRenderTemplate)
+	rolebinding := &rbacv1.ClusterRoleBinding{}
+	if err := yaml.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(renderedInfraRoleBinding)), 500).Decode(&rolebinding); err != nil {
+		panic(err)
+	}
+
+	return []crclient.Object{serviceaccount, role, rolebinding}
+}
+
+func GenerateKubeconfig(serverUrl string, namespace string, certificateAuthorityData string, userToken string) string {
+	mustacheRenderTemplate := map[string]string{
+		"SERVER_URL":                 serverUrl,
+		"NAMESPACE":                  namespace,
+		"CERTIFICATE_AUTHORITY_DATA": certificateAuthorityData,
+		"USER_TOKEN":                 userToken,
+	}
```

Comment:
i don't think this is being used any more now that we're building the kubeconfig using the hypershift helper functions.